test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -204,7 +202,8 @@ class cursor_bound_01 : public test_harness::test {
             // Normal case
             // lower_key = std::string("0");
             range_cursor->set_key(range_cursor.get(), lower_key.c_str());
-            range_cursor->bound(range_cursor.get(), "bound=lower");
+            ret = range_cursor->bound(range_cursor.get(), "bound=lower");
+            testutil_check(ret == 0 || ret == EINVAL);
```

Comment:
`EINVAL` means the lower bound could not be set? Same Q for the upper bound.

Reply:
In the case that we get an EINVAL for a bound, the lower bound will not be reset into a different value. However this doesn't mean that the bounds wouldn't be set at all. It depends if the cursor has already set a bound prior to this.